test/karma/tsconfig.json

```diff
@@ -26,7 +26,8 @@
     "paths": {
       "@stencil/core": ["../../internal"],
       "@stencil/core/internal": ["../../internal"],
-      "@stencil/core/testing": ["../../testing"]
+      "@stencil/core/testing": ["../../testing"],
+      "@test-invisible-prehydration": ["./test-invisible-prehydration"]
```

Comment:
Do we use this path anywhere in the code? I don't see `@test-invisible-prehydration` anywhere in this PR